modules/ROOT/pages/import-api-specification.adoc

```diff
@@ -2,7 +2,7 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:page-aliases: 7.4@studio::import-api-secification.adoc
+:page-aliases: 7.4@studio::import-api-specifications.adoc
```

Comment:
I don't know if this was an intentional typo, since import-api-specification is the name of this adoc file and doesn't need an alias. I added a different alias - please tell me if i need to revert.

Reply:
The page name takes no "s" at the end. ":page-aliases: 7.4@studio::import-api-secification.adoc" is OK, but I don't really understand the function of this "page-aliases".

Reply:
hi @Crispy-Salesforce, thanks for asking for clarification on this. The page-aliases is a URL redirect, so with the current site, if you open https://docs.mulesoft.com/studio/7.4/import-api-secification (with the typo in the secification), it will redirect to the actual page URL which is https://docs.mulesoft.com/studio/7.4/import-api-specification.

After fixing this typo, we actually don't need this page-alias because the name is the same as the page's name. The new suggested page alias (https://docs.mulesoft.com/studio/7.4/import-api-specifications), currently goes to the 404 not found page, will instead redirect to this page. I don't have the historical context for why the original misspelled alias exists - we can keep it there if needed, or update it - totally your call at this point.

Reply:
Hi, Gary!
If it will redirects to the appropriate page, let it be! hahaha
Thanks for the explanation!